packages/forms/src/UIForm/fields/CheckBox/SimpleCheckBox.component.js

```diff
@@ -11,6 +11,7 @@ export default function SimpleCheckBox({ id, label, onChange, schema, value }) {
 					autoFocus={autoFocus}
 					disabled={disabled}
 					label={label}
+					onBlur={event => onFinish(event, { schema })}
```

Comment:
Why did the onBlur got renamed to onFinish?

Reply:
Because it's a widget props. When user finish to change value, this is triggered. In checkbox, it is on blur, so if the checkbox is changed multiple time, the onFinish will be triggered once, on blur

Reply:
OK. So there is a clear difference between onBlur and onFinish. 
Sounds good.